### ISSUE TITLE
3.2.0-alpha.6: cannot save section settings for singles w/ versioning on

### ISSUE THREAD

**Issue by timkelty** [craftcms/cms#4303]
This happens when saving section settings for a single, or saving a single's entry type settings.
Unchecking "Enable versioning for entries in this section?" allows it to proceed. 

```
Argument 2 passed to craft\services\Revisions::createRevision() must be of the type int, null given, called in /app/vendor/composer/craftcms/cms/src/elements/Entry.php on line 1110
```

Argument 2 here is `$currentEntry->authorId`.

**Comment by brandonkelly** [comment]
Thanks, just fixed this for the next release.

**Comment by brandonkelly** [comment]
And just released Alpha 6.3 with the fix.
